tests/rbd/test_rbd_image_zero_size.py

```diff
@@ -0,0 +1,115 @@
+"""Module to verify resize operation of RBD image with zero size.
+
+Test case covered -
+CEPH-83597243 -  Test to verify RBD image with zero size
+
+Pre-requisites :
+1. Cluster must be up and running with capacity to create pool
+2. We need atleast one client node with ceph-common package,
+   conf and keyring files
+
+Test Case Flow:
+1. Create a pool with name as 'images'
+2. Create an image inside this pool with 0 size' (rbd create images/image_zero1 --size 0)
+3. Create another image inside this pool with size 1024 (rbd create images/image_1024size --size 1024)
+4. Use rbd du command to check that image size for both images
+5. Resize second image size to 0
+6. Remove that images
+
+"""
+
+from ceph.parallel import parallel
+from tests.rbd.exceptions import RbdBaseException
+from tests.rbd.rbd_utils import (
```

Comment:
`rbd_utils` got deprecated, can you please use `initial_rbd_config` from from `ceph.rbd.initial_config import initial_rbd_config`

Reply:
Replaced from tests.rbd.rbd_utils with "from ceph.rbd.initial_config import initial_rbd_config"